content/terraform/v1.13.x/docs/language/meta-arguments/depends_on.mdx

```diff
@@ -0,0 +1,195 @@
+
+## Supported constructs
+
+You can use `depends_on` in the following Terraform configuration blocks:
+
+- [`check` blocks](/terraform/language/block/resource)
+- [`data` blocks](/terraform/language/block/data)
+- [`ephemeral` blocks](/terraform/language/block/ephemeral)
+- [`module` blocks](/terraform/language/block/module)
+- [`output` blocks](/terraform/language/block/output)
+- [`resource` blocks](/terraform/language/block/resource)
+
+## Example use cases 
+
+The following use cases describe common patterns for the `depends_on` argument.
+
+
+### Specify a dependency during validation
+
+Add a `depends_on` argument to a `data` block nested in a [`check` block](/terraform/language/block/check) validation when the data source depends on another resource but doesn't reference the resource directly.  
+
+`check` blocks validate infrastructure outside of the typical resource lifecycle. You can nest a `data` block inside a `check` block to fetch information that you can reference in an `assert` block. By adding a `depends_on` argument to the `data` block nested in a `check` block requires, Terraform retrieves the necessary data before running the validation.
+
+For example, if you define a `check` that verifies that a website API returns `200`, that check fails the first time Terraform runs your configuration because your website's infrastructure does not exist yet. You can set the `depends_on` argument to a resource, such as the load balancer, to ensure Terraform only runs the `check` once the website is up. When running an operation, Terraform evaluates the `check`, warns `known after apply` until that crucial piece of your website is ready, and continues the operation.
+
+However, this strategy only works when the `data` block does not directly reference the resource specified in the `depends_on` argument. Otherwise, anytime that resource changes, the `check` block warns `known after apply` until Terraform updates that resource, making your check potentially noisy and ineffective. 
+
+In the following example, Terraform waits to run the check until it creates the `aws_db_instance.main` database. Terraform prints `known after apply`, instead of printing false warnings, until it finishes creating the database:
+
+```hcl
+check "database_connection" {
+ data "postgresql_database" "app_db" {
+   name       = "application"
+   depends_on = [aws_db_instance.main]
+ }
+
+ assert {
+   condition     = data.postgresql_database.app_db.allow_connections
+   error_message = "Database is not accepting connections"
+ }
+}
+```
+
+### Create a database service that other services depend on 
+
+In the following example, Terraform must launch the `aws_db_instance`  service first so that `aws_instance` can connect to it:
```

Comment:
```suggestion
In the following example, Terraform must launch the `aws_db_instance` service first so that `aws_instance` can connect to it:
```

Extra space